packages/now-cli/test/fixtures/unit/now-dev-next-rewrites/now.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": 2,
+  "name": "now-dev-next-rewrites",
+  "builds": [{ "src": "package.json", "use": "@now/next@canary" }],
```

Comment:
```suggestion
  "builds": [{ "src": "package.json", "use": "@vercel/next@canary" }],
```

Reply:
you may want to have a look at the other fixtures, quite a few of them are still referencing `@now` builders